pkg/netcertmanager/certificate.go

```diff
@@ -0,0 +1,272 @@
+/*
```

Comment:
I assume all the files from net-certmanager were just copied over, right?

Reply:
Yes should I update the date in the heard or something else?

Reply:
No, just wanted to make sure, so people know not to really in-depth review them.